### DISCUSSION TITLE
表格识别，错误的行合并

### DISCUSSION THREAD

**Discussion by archeradvanture** [PaddlePaddle/PaddleOCR#12942] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：MacOS
- 版本号/Version：Paddle： 2.5.2     PaddleOCR： 2.7.0.3  问题相关组件/Related components： PP-structure
- 运行指令/Command Code：
- 完整报错/Complete Error Message：

我们提供了AceIssueSolver来帮助你解答问题，你是否想要它来解答(请填写yes/no)?/We provide AceIssueSolver to solve issues, do you want it? (Please write yes/no): yes

表格识别时，最后一行合并到上一行的某个单元格内。 

模型使用的是 ch_ppstructure_mobile_v2.0_SLANet_infer

![image](https://github.com/PaddlePaddle/PaddleOCR/assets/146340308/a00234a1-126e-403c-b61c-df58af67bfd1)

![image](https://github.com/PaddlePaddle/PaddleOCR/assets/146340308/9f2a828c-d544-49b1-86f8-7391ff9a5fe6)

**Comment by gl94** [comment]
这是由于表格单元格预测错误导致的,可以单独可视化ch_ppstructure_mobile_v2.0_SLANet_infer的单元格检测效果

**Comment by archeradvanture** [comment]
> 这是由于表格单元格预测错误导致的,可以单独可视化ch_ppstructure_mobile_v2.0_SLANet_infer的单元格检测效果

抱歉没太懂什么意思，有具体怎么做的方法吗？

**Comment by gl94** [comment]
> > 这是由于表格单元格预测错误导致的,可以单独可视化ch_ppstructure_mobile_v2.0_SLANet_infer的单元格检测效果
> 
> 抱歉没太懂什么意思，有具体怎么做的方法吗？

python3 tools/infer_table.py -c configs/table/SLANet_ch.yml -o Global.pretrained_model=./pretrained/ch_ppstructure_mobile_v2.0_SLANet_train/best_accuracy  Global.infer_img=你的图片文件夹路径 可以在output中看到单元格框的结果

**Comment by gl94** [comment]
同时注意到你用的模型是针对纯表格的，可以对原图进行裁切，去掉表格之外的部分后进行推理
